src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
```

Comment:
Add an empty private constructor in this class. 
UPD: This is written in the checklist.

Reply:
> Add an empty private constructor in this class.
> UPD: This is written in the checklist.

\+